consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/BaseBftProtocolSchedule.java

```diff
@@ -106,22 +97,9 @@ private ProtocolSpecBuilder applyBftChanges(
         .blockImporterBuilder(MainnetBlockImporter::new)
         .difficultyCalculator((time, parent, protocolContext) -> BigInteger.ONE)
         .skipZeroBlockRewards(true)
-        .blockHeaderFunctions(BftBlockHeaderFunctions.forOnchainBlock(bftExtraDataCodec));
-
-    blockReward.ifPresent(bigInteger -> builder.blockReward(Wei.of(bigInteger)));
-
-    if (configOptions.getMiningBeneficiary().isPresent()) {
-      final Address miningBeneficiary;
-      try {
-        // Precalculate beneficiary to ensure string is valid now, rather than on lambda execution.
-        miningBeneficiary = Address.fromHexString(configOptions.getMiningBeneficiary().get());
```

Comment:
The `miningBeneficiary` is now stored as an `Address`, so we don't have to interpret it here.